CMakeLists.txt

```diff
@@ -132,22 +132,8 @@ endif()
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/natives/${NATIVE_LIB_DIR})
 
 # version.h using #cmakedefine for version from pom.xml.
-set(JSSC_VERSION "0.0.0-UNKNOWN")
-file(STRINGS ${CMAKE_CURRENT_SOURCE_DIR}/pom.xml POM_FILE)
-foreach(POM_LINE ${POM_FILE})
-	# Assume first "<version>" is the project version
-	if(POM_LINE MATCHES "<version>")
-		string(REGEX REPLACE "^[ \t]+|<[^>]*>" "" DETECTED_VERSION "${POM_LINE}")
-		string(STRIP "${DETECTED_VERSION}" DETECTED_VERSION)
-		if(DETECTED_VERSION STREQUAL "")
-			message(WARNING "Could not parse JSSC version from pom.xml, defaulting to \"${JSSC_VERSION}\"")
-		else()
-			set(JSSC_VERSION "${DETECTED_VERSION}")
-			message(STATUS "Found JSSC version \"${JSSC_VERSION}\" in pom.xml")
-		endif()
-		break()
-	endif()
-endforeach()
+message(STATUS "Release JSSC version based on pom.xml: \"$ENV{JSSC_VERSION}\"")
+set(JSSC_VERSION "$ENV{JSSC_VERSION}")
```

Comment:
This was problematic, as the version extracted from pom.xml this way may be a `-SNAPSHOT` version - and we do not want to hardcode this into the binary. We'll get this value from the environment, and let maven calculate it.

Reply:
It's only problematic if it's provided by maven.  Please restore set this value conditionally.  More details here: https://github.com/java-native/jssc/pull/102#commitcomment-56306951